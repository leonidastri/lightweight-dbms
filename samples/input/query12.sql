select distinct S.A, B.E from Sailors S, Boats B, Reserves R;