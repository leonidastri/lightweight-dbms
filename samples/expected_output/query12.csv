1,2
1,3
1,104
1,1
2,2
2,3
2,104
2,1
3,2
3,3
3,104
3,1
4,2
4,3
4,104
4,1
5,2
5,3
5,104
5,1
6,2
6,3
6,104
6,1